tests/integration/test_events.py

```diff
@@ -431,6 +432,49 @@ def test_put_events_with_target_lambda(self):
             Name=bus_name
         )
 
+    def test_rule_disable(self):
+
+        rule_name = 'rule-{}'.format(short_uid())
+        self.events_client.put_rule(
+            Name=rule_name,
+            ScheduleExpression='rate(1 minutes)'
+        )
+
+        queue_name = 'queue-{}'.format(short_uid())
+        queue_target_id = short_uid()
+        queue_url = self.sqs_client.create_queue(QueueName=queue_name)['QueueUrl']
+        queue_arn = aws_stack.sqs_queue_arn(queue_name)
+
+        event = {
+            'env': 'testing'
+        }
+
+        self.events_client.put_targets(
+            Rule=rule_name,
+            Targets=[
+                {
+                    'Id': queue_target_id,
+                    'Arn': queue_arn,
+                    'Input': json.dumps(event)
+                }]
+        )
+
+        response = self.events_client.list_rules()
+        self.assertEqual(response['Rules'][0]['State'], 'ENABLED')
+
+        self.events_client.disable_rule(Name=rule_name)
+
+        response = self.events_client.list_rules()
+        self.assertEqual(response['Rules'][0]['State'], 'DISABLED')
+        time.sleep(62)
```

Comment:
I think we should remove lines 469-471 here - probably not worth increasing the test time by more than a minute here (we can rely on the `State` attribute to assert that the rule has been disabled).

Reply:
The actual issue is that even though the state changes, the scheduler still runs. So if we remove the sleep, the test will pass without the fix as well. To test that it doesn't run, we have to wait for the expression interval. Is there any way we can reduce the time?